### PR TITLE
[DEV-10317] Update domestic place_of_performance ES query [hotfix][qat] 

### PR DESCRIPTION
**Pull request by aguest-kc**
Duplicate of #3941 for QAT

### CODE REVIEW

**Comment by aguest-kc** [comment]
Don't merge QAT into the new branch until the Staging and Master PRs have been merged.

**Review by dporth-frb — APPROVED** [review]

**Pull request merged**
